plugins/PrivacyManager/vue/src/Compliance/ComplianceOverview.vue

```diff
@@ -0,0 +1,101 @@
+<!--
+  Matomo - free/libre analytics platform
+
+  @link    https://matomo.org
+  @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <ContentBlock :content-title="title">
+    <p>{{ description }}</p>
+    <ComplianceTable
+      v-if="!state.loading"
+      :results="state.compliance_indicators"
+    />
+    <ActivityIndicator :loading="state.loading"/>
+    <Field
+      v-if="!state.loading"
+      uicontrol="checkbox"
+      name="enableFeature"
+      :title="translate('PrivacyManager_ComplianceEnforceCheckboxIntro')"
+      :introduction="translate('PrivacyManager_ComplianceEnforceCheckboxTitle')"
+      :inline-help="translate('PrivacyManager_ComplianceEnforceCheckboxHelp')"
+      v-model="isComplianceModeEnabled"
```

Comment:
Possibly rename so that it indicates better what it would do?
```suggestion
      v-model="shouldEnforceComplianceMode"
```

Reply:
renamed